<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initListDialog</class>
 <widget class="QDialog" name="initListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New List</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string notr="true">Points Limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pointsLimitSpin">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="ucmRadio">
      <property name="text">
       <string notr="true">UCM</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">factionRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="scourgeRadio">
      <property name="text">
       <string notr="true">Scourge</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">factionRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="phrRadio">
      <property name="text">
       <string notr="true">PHR</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">factionRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="shaltariRadio">
      <property name="text">
       <string notr="true">Shaltari</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">factionRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="resistanceRadio">
      <property name="text">
       <string notr="true">Resistance</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">factionRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string notr="true">New List</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="factionRadioGroup"/>
 </buttongroups>
</ui>
